resources/build-integration.md

```diff
@@ -21,6 +21,51 @@ We recommend checking out the READMEs of our other integration repositories, lik
 
 All integration repositories must contain a file titled `CONTRIBUTING.md`. The template is [here](/templates/CONTRIBUTING.md) and should be filled according to the details of your repository..
 
+### 3.1 Docker & Docker-compose
+
+The integration should be ready to use with `Docker`, and it must contain a `Docker Compose` file configurated following the patterns of the other repositories.
+- Must have a service `package` and a `meilisearch` service.
+  - Must depend and contain a link to `meilisearch` service.
```

Comment:
In the case I understood it correctly I would have put the two essential services in the explanation:
```
- `meilisearch` Must contain a link to the official Meilisearch docker image.
- `package` Must depend on `meilisearch` and build a local image of the package.
```

Reply:
Then it's the indentation that has a problem because I didn't read it like that at all:
```
- Must have a service `package` and a `meilisearch` service.
  - Must depend and contain a link to `meilisearch` service.
  - The `meilisearch` service should point to `latest` tag.
    - `MEILI_NO_ANALYTICS` should be set to `true`
    - `MEILI_MASTER_KEY` should be set to `masterKey`
```

Reply:
I've approved it so that it can be merged. I don't think it's a significant problem, but it's still unclear the way it is